crypto/src/main/java/net/consensys/cava/crypto/sodium/Signature.java

```diff
@@ -502,4 +526,49 @@ public static boolean verifyDetached(byte[] message, byte[] signature, Signature
 
     return true;
   }
+
+  public static Bytes sign(Bytes message, Signature.SecretKey secretKey) {
```

Comment:
can you copy the javadoc from the method below here too please?

Reply:
javadoc added